pkg/gofr/migration/sql.go

```diff
@@ -17,11 +17,31 @@ const (
     constraint primary_key primary key (version, method)
 );`
 
+	createSQLGoFrMigrationsTableMSSQL = `IF NOT EXISTS (SELECT * FROM sys.objects 
+	WHERE object_id = OBJECT_ID(N'gofr_migrations') AND type = N'U')
+    BEGIN
+        CREATE TABLE gofr_migrations (
+                                         version BIGINT NOT NULL,
+                                         method VARCHAR(4) NOT NULL,
+                                         start_time DATETIME2 NOT NULL,
+                                         duration BIGINT,
+                                         CONSTRAINT PK_gofr_migrations PRIMARY KEY (version, method)
+        );
+    END;
+`
+
 	getLastSQLGoFrMigration = `SELECT COALESCE(MAX(version), 0) FROM gofr_migrations;`
 
 	insertGoFrMigrationRowMySQL = `INSERT INTO gofr_migrations (version, method, start_time,duration) VALUES (?, ?, ?, ?);`
 
 	insertGoFrMigrationRowPostgres = `INSERT INTO gofr_migrations (version, method, start_time,duration) VALUES ($1, $2, $3, $4);`
+
+	insertGoFrMigrationRowMSSQL = `INSERT INTO gofr_migrations (version, method, start_time, duration) VALUES (?, ?, ?, ?);`
+
+	mySQL    = "mysql"
+	mssql    = "mssql"
+	sqlite   = "sqlite"
+	postgres = "postgres"
```

Comment:
I have renamed them to `dialectMySQL ` etc. but moving them to `pkg/gofr/datasource/sql` doesn't seem right because they are mostly being used inside the migration directory only.

Reply:
Not exactly

https://github.com/gofr-dev/gofr/blob/development/pkg%2Fgofr%2Fdatasource%2Fsql%2Fbind.go#L9

Reply:
Bit this could be addressed in another PR